### PR TITLE
The TC Inflation Act of 2023 - No Balance Changes This Time

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Multiplies most TC costs, rewards, and totals by five.
`20TC` Traitor Uplinks > `100TC` etc etc etc.
Contractor payouts and totals multiplied by `5`.
Nuclear Operatives total TC multiplied by `5`.
Uplink Items and such all multiplied by `5`.

## Why It's Good For The Game
This has been discussed by most of the Dev Team for a few months now. This will allow for more variance in Uplink/Operative items in the future, as well as allow us to fine-tune items that are over/underperforming in a more slight way.

This will lead to people having to add bigger numbers together. I do hope everyone is capable of basic math. If not, your phone or computer has a calculator if you cant count to 100 (Operatives on the other hand get a LOT more TC to work with, so I wont fault you there).

In the future, we've discussed adding a "checkout" system, especially for Operatives, so that you can add stuff to a "cart" and check out your items in bulk, perhaps with some discounts that way for Traitors too. But, as that touches TGUI and would build upon this, Ill leave that for later/someone else to do it. 

## Testing
1. Spawn in.
2. Purchase warcrimes from various uplinks.
3. Double check that Contracts are paid out correctly.

## Changelog
:cl:
tweak: Tweaked TC economy to work from a 100TC budget for Traitors and radiating changes to all other Uplink systems.
/:cl:

### CODE REVIEW

File: code/datums/uplink_items/uplink_nuclear.dm [hunk 1]
```diff
@@ -157,132 +157,132 @@
...
-	cost = 2
+	cost = 10
 	uplinktypes = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST)
 
 /datum/uplink_item/ammo/bulldog_ammobag
 	name = "Bulldog - 12g Ammo Duffel Bag"
 	desc = "A duffel bag filled with enough 12g ammo to supply an entire team, at a discounted price."
 	reference = "12ADB"
 	item = /obj/item/storage/backpack/duffel/syndie/shotgun
-	cost = 12 // normally 18
```
Comment (hal9000PR) [inline code/datums/uplink_items/uplink_nuclear.dm:224]: 12 * 5 = 60 not 75

File: code/datums/uplink_items/uplink_nuclear.dm [hunk 2]
```diff
@@ -157,132 +157,132 @@
...
+	cost = 75 // normally 90
 	uplinktypes = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST)
 
 /datum/uplink_item/ammo/bulldog_XLmagsbag
 	name = "Bulldog - 12g XL Magazine Duffel Bag"
 	desc = "A duffel bag containing three 16 round drum magazines(Slug, Buckshot, Dragon's Breath)."
 	reference = "12XLDB"
 	item = /obj/item/storage/backpack/duffel/syndie/shotgunXLmags
-	cost = 12 // normally 18
```
Comment (hal9000PR) [inline code/datums/uplink_items/uplink_nuclear.dm:232]: 12 * 5 is 60

**Review by hal9000PR — CHANGES_REQUESTED** [review]

**Review by hal9000PR — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]
MLGTM

**Review by DGamerL — APPROVED** [review]
The math seems to be mathing

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**